src/main.rs

```diff
@@ -93,13 +104,16 @@ fn run(args: Cli) -> Result<i32> {
         return Ok(0);
     }
 
+    // Maybe we can use log::warn!() for this instead?
```

Comment:
Sounds good. This should also make implementing https://github.com/ducaale/xh/issues/179 much easier 👍